docs/_Docs_Colony.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Colony
+section: Docs
+order: 0
+---
+
+Colony is a protocol for a new type of open, meritocratic organization that operates via software rather than paperwork and management hierarchy.
+
+A colony is a set of smart contracts on the Ethereum blockchain that contains all of the normal functions of a traditional firm, as well as some new functions that are only possible using a decentralized platform. Using the functions built into Colony, an organization can do:
+
+* Decision making
+* Ownership
+* Reputation
+* Dispute resolution
+* Work management and delegation
+* Financial management
+
+The basic ideas of how Colony fits together are presented below.
+
+==TOC==
+
+## `Tasks`
+The smallest conceptual unit within a Colony is a **task**. A task is a discrete unit of work which requires no further subdivision or delegation, and which can be evaluated as complete or incomplete based on some set of criteria.
+
+There is intentionally no further prescription for how a task is meant to be used within a colony. Depending on context and criteria, a task could be called a "bounty", a "salary", a "reimbursement", or an "incentive".
+
+At a minimum, a newly created task must be assigned a `domainId` and reference a specification for the task's completion, i.e. a description of the work to be done and how that work will be evaluated.
+
+[More about Tasks](/colonynetwork/docs-tasks/)
+
+
+## `Reputation and Tokens`
+In an organization of any type, reputation is an essential heuristic for people to keep track of each other's perceived merit in an environment of limited information. Reputation within Colony is meant stay as close as possible to the concept of reputation as it is commonly understood. Reputation is a representation of _merit_, which in Colony implies an immutable record of contributions to a shared goal.
+
+In the Colony Protocol, **reputation** is a number that quantifies a particular individual’s influence, calculated from the sum of work that has been completed within the colony.
+
+Every Colony has its own **native token** which complements reputation. Tokens, when earned as a task payout, create reputation for the recipient at a 1:1 ratio.
```

Comment:
The token balance is part of the calculation and reputation earned is proportionate to it, but not 1:1. The other key part is the result of the performance ratings of the task work

Reply:
Shoot. Yes, I believe I changed this in the reputation page but not here; thanks, will revise